File: easy-solution/minesp3164/349.py
class Solution(object):
    def intersection(self, nums1, nums2):
        set1 = set(nums1)
        set2 = set(nums2)
        if len(set1) > len(set2):
            return set1.intersection(set2)
        else:
            return set2.intersection(set1)

File: easy-solution/galaxy4276/349.java
import java.util.*;

class Solution {
  public int[] intersection(int[] nums1, int[] nums2) {
    var list = new ArrayList<Integer>();
    var set = new HashSet<Integer>();
    for (int i : nums1) list.add(i);
    for (int j : nums2)
      if (list.contains(j))
        set.add(j);
    int[] arr = Stream.of(set.toArray(new Integer[set.size()])).mapToInt(Integer::intValue).toArray();
    return arr;
  }
}
